bot.py

```
@@ -81,6 +82,18 @@ async def initialise_db():
         'INTEGER, message_id INTEGER)'
     )
 
+    try:
+        await cur.execute(
+            'ALTER TABLE verification_codes ADD COLUMN generated_at TIMESTAMP')
+
+        await cur.execute(
+            "UPDATE verification_codes SET generated_at = datetime('now') WHERE generated_at IS NULL;")
+    except Exception as e:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)